hokusai/production.yml

```diff
@@ -0,0 +1,105 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: artsy-wwwify-web
+  namespace: default
+spec:
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: artsy-wwwify
+        component: web
+        layer: application
+      name: artsy-wwwify-web
+    spec:
+      containers:
+        - name: artsy-wwwify-web
+          envFrom:
+            - configMapRef:
+                name: artsy-wwwify-environment
+          image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/artsy-wwwify:production
+          imagePullPolicy: Always
+          ports:
+            - name: wwwify-http
+              containerPort: 3000
```

Comment:
Why `3000`? Usually we use `8080`.

Reply:
This app doesn't use nginx (so there's no `80`/`8080` parity) and doesn't currently depend on any config. I could add all that but didn't think the complexity was worth it, as the whole app is only a few lines of code. I can update this to use `8080` instead and the load-balancer to proxy to that, but since it's still different than the many other apps using nginx that might just be confusing.